metrics/web/lib/common/presentation/navigation/metrics_page/metrics_page.dart

```diff
@@ -21,15 +25,17 @@ class MetricsPage<T> extends Page<T> {
   /// The [maintainState] defaults to `true`.
   /// The [fullscreenDialog] defaults to `false`.
   ///
-  /// The [child], [fullscreenDialog], and [maintainState] must not be null.
+  /// The [child], [routeName], [fullscreenDialog], and [maintainState] must not be null.
   const MetricsPage({
     @required this.child,
+    @required this.routeName,
     this.maintainState = true,
     this.fullscreenDialog = false,
     LocalKey key,
     String name,
```

Comment:
🔨 
What do you think about avoiding passing this value to the constructor and overriding the `name` field instead?